README.md

```diff
@@ -1,2 +1,74 @@
 # Saga [![Build Status](https://travis-ci.org/ServiceComb/saga.svg?branch=master)](https://travis-ci.org/ServiceComb/saga?branch=master) [![Coverage Status](https://coveralls.io/repos/github/ServiceComb/saga/badge.svg?branch=master)](https://coveralls.io/github/ServiceComb/saga?branch=master) [![License](https://img.shields.io/badge/license-Apache%202-4EB1BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
 
+## Purpose
+Saga is implemented based on the compensation transaction driven mechanism, which provides a simple way to help users solve the data consistency problems encountered in micro-service applications.
+
+## Documentation
+Reference documentation is available on the [ServiceComb website][servicecomb-website].
+
+[servicecomb-website]: http://servicecomb.io/
+
+## Major Architecture of Saga
+* saga-core(transaction and compensation handled)
+* saga-format(events serialized and deserialized)
+* saga-transports(protocol processing as rest and rpc)
+* saga-discovery(discovery service center)
+* saga-spring(running architecture)
```

Comment:
* saga-core (transaction and compensation handling logic)
* saga-format (data serialization and deserialization)
* saga-transports (communication protocol implementation such as rest or rpc in the future)
* saga-discovery (service discovery)
* saga-spring (restful service framework)

Reply:
modified completed